frontend/src/metabase/lib/urls.js

```diff
@@ -11,8 +11,13 @@ export function question(cardId, hash = "", query = "") {
     }
     if (query && typeof query === "object") {
         query = Object.entries(query)
-            .map(kv => kv.map(encodeURIComponent).join("="))
-            .join("&");
+            .map(kv => {
+                if (Array.isArray(kv[1])) {
+                    return kv[1].map(v => `${kv[0]}=${v}`).join('&');
```

Comment:
I think `kv[0]` and `v` need to be `encodeURIComponent`'d

Reply:
Also some tests with values that include special characters like `&` and `=` would be good.